middleware/locals.js

```diff
@@ -20,6 +22,49 @@ module.exports = {
     middleware: function(req, res, next) {
         const locale = req.i18n.getLocale();
 
+        /**
+         * Allows feature flags to be passed through as query strings
+         * e.g. ?feature=use-new-header
+         * Useful for testing new features
+         */
+        res.locals.queryFeature = function(name) {
+            const featureNames = ['use-new-header'];
```

Comment:
Maybe this should live in a config file so we don't have this string in multiple places?